### DISCUSSION TITLE
Feature Request: XY Chart Time Series Units and Axis Scaling

### DISCUSSION THREAD

**Discussion by blakeroberts-wk** [grafana/grafana#66073] [Dashboards, Panels & Visualizations]
The XY Chart should support displaying custom units and axis scaling for both the x and y axis.

A relatively common use case for both of these features is in tail latency graphs.

<img width="903" alt="image" src="https://user-images.githubusercontent.com/28708077/230225217-08c62b67-29e4-4147-b86f-7fc4fcc679c4.png">

In this example, the x axis units should be displayed as a percentage and scale non-linearly to "zoom" into the "tail" latency (90% - 99.99...%) and the y axis units should display in milliseconds. Scaling the y axis non-linearly may have similar benefits.

Epic related to the XY Chart feature release: https://github.com/grafana/grafana/issues/61734

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82786. Feel free to continue the discussion around this there. Thank you!
